apps/vr-tests/package.json

```diff
@@ -24,8 +24,8 @@
     "@fluentui/react": "^8.34.5",
     "@fluentui/react-accordion": "9.0.0-alpha.80",
     "@fluentui/react-avatar": "9.0.0-alpha.84",
-    "@fluentui/react-button": "^9.0.0-alpha.90",
-    "@fluentui/react-card": "^9.0.0-alpha.6",
+    "@fluentui/react-button": "9.0.0-alpha.90",
```

Comment:
@andrefcdias  are you going to PR this into master first ?

Reply:
Fixed on #19903